tests/test_prtvol2csv.py

```diff
@@ -183,6 +183,23 @@ def test_prtvol2csv(tmp_path, mocker):
     pd.testing.assert_frame_equal(dframe, expected)
 
 
+def test_correct_parsing_date(tmp_path):
+    shutil.copy(TEST_PRT_DATADIR / "DROGON_FIPNUM.PRT", tmp_path / "DROGON_FIPNUM.PRT")
+    os.chdir(tmp_path)
+    # Replace "REPORT   0     1 JAN 2018" with "REPORT   0     5 JLY 2018"
+    prt_path = tmp_path / "DROGON_FIPNUM.PRT"
+    prt_text = prt_path.read_text(encoding="utf8")
+    prt_text = prt_text.replace(
```

Comment:
This is a slightly unsafe test, because if the `replace() `function fails to find your search string, `prt_text` will go through unmodified and no error/warning emitted. So if the underlying PRT file changes, this test will become defunct, but noone will notice. 

If you first assert that the string you are replacing is present, then you will avoid that problem.